ui/components/app/flask/snap-settings-card/snap-settings-card.js

```diff
@@ -55,36 +57,6 @@ const SnapSettingsCard = ({
   chipProps,
 }) => {
   const t = useI18nContext();
-  const [chipStatus, setChipStatus] = useState(STATUSES.INSTALLING);
```

Comment:
We could split these changes out, since this component is on `develop` already

Reply:
@shanejonas